parts/tools/check_frame.py

```
@@ -0,0 +1,384 @@
+    src = ('use <case_polykybd_split72_lr.scad>\n'
+           'use <../parts/diffuser_frame_left.scad>\n'
+           f'intersection() {{ right_spacer(); {placement} diffuser_frame_left(); }}\n')
+    # Must live in CASE: `use <case_polykybd_split72_lr.scad>` resolves relative
+    # to the scad file, not the cwd.  Unique name so two runs cannot delete each
+    # other's input.
+    fd, sf = tempfile.mkstemp(prefix='_check_tmp_', suffix='.scad', dir=CASE)
+    with os.fdopen(fd, 'w') as fh:
+        fh.write(src)
+    try:
+        of = os.path.join(workdir, 'x.stl')
+        r = subprocess.run(['openscad', '-o', of, '--export-format', 'asciistl', sf],
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)